<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>311</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Ajouter Client</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Pr√©nom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Age</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>541</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>identification</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>MDP:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>200</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 255);</string>
      </property>
      <property name="text">
       <string>log in</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>receipt</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>541</width>
        <height>381</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>new</string>
       </attribute>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>290</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>pick date</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>purchases</string>
        </property>
       </widget>
       <widget class="QComboBox" name="achat_list">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>20</y>
          <width>81</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="currentText">
         <string>products</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>160</y>
          <width>83</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>paid</string>
        </property>
       </widget>
       <widget class="QPushButton" name="save_receipt">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>180</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 255);</string>
        </property>
        <property name="text">
         <string>save receipt</string>
        </property>
       </widget>
       <widget class="QDateTimeEdit" name="date_time">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>290</y>
          <width>194</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>60</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>quantity</string>
        </property>
       </widget>
       <widget class="QPushButton" name="ajouter_button">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>60</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 255);</string>
        </property>
        <property name="text">
         <string>add to receipt</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_id_client">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>170</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="placeholderText">
         <string>id client</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_supprimer">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>100</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>delete</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_reference">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>140</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>reference</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_montant">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>200</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>montant</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_caissier">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>230</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>caissier</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_6">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>260</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>achats</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>edit</string>
       </attribute>
       <widget class="QTabWidget" name="tabWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>531</width>
          <height>341</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tabWidgetPage1">
         <attribute name="title">
          <string>1</string>
         </attribute>
         <widget class="QLabel" name="label_8">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>10</y>
            <width>71</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>find receipt</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_7">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>40</y>
            <width>113</width>
            <height>22</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_11">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>81</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>receipt date:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_8">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>80</y>
            <width>113</width>
            <height>22</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">
background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_12">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>80</y>
            <width>71</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>receipt n¬∞:</string>
          </property>
         </widget>
         <widget class="QComboBox" name="list_facture">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>150</y>
            <width>291</width>
            <height>22</height>
           </rect>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="currentText">
           <string>list of receipts</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_4">
          <property name="geometry">
           <rect>
            <x>230</x>
            <y>110</y>
            <width>101</width>
            <height>28</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 255);</string>
          </property>
          <property name="text">
           <string>search</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_5">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>180</y>
            <width>93</width>
            <height>28</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 255);</string>
          </property>
          <property name="text">
           <string>open</string>
          </property>
         </widget>
         <widget class="QPushButton" name="delete_button">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>180</y>
            <width>93</width>
            <height>28</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 255);</string>
          </property>
          <property name="text">
           <string>delete</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_7">
          <property name="geometry">
           <rect>
            <x>240</x>
            <y>180</y>
            <width>93</width>
            <height>28</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 255);</string>
          </property>
          <property name="text">
           <string>notify client</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tabWidgetPage2">
         <attribute name="title">
          <string>2</string>
         </attribute>
         <widget class="QLabel" name="label_13">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>20</y>
            <width>56</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>receipt:</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_8">
          <property name="geometry">
           <rect>
            <x>180</x>
            <y>160</y>
            <width>93</width>
            <height>28</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 255);</string>
          </property>
          <property name="text">
           <string>print</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pb_modifier">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>160</y>
            <width>93</width>
            <height>28</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 255);</string>
          </property>
          <property name="text">
           <string>edit</string>
          </property>
         </widget>
         <widget class="QTableView" name="facture_affiche">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>0</y>
            <width>311</width>
            <height>151</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>data</string>
       </attribute>
       <widget class="QToolBox" name="toolBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>531</width>
          <height>341</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page_5">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>531</width>
           <height>281</height>
          </rect>
         </property>
         <attribute name="label">
          <string>stats</string>
         </attribute>
         <widget class="QPushButton" name="statsmontant">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>60</y>
            <width>121</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>selon le montant</string>
          </property>
         </widget>
         <widget class="QTableView" name="facture_affiche_2">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>110</y>
            <width>311</width>
            <height>151</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_9">
          <property name="geometry">
           <rect>
            <x>390</x>
            <y>170</y>
            <width>113</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QRadioButton" name="radioButton_2">
          <property name="geometry">
           <rect>
            <x>380</x>
            <y>110</y>
            <width>97</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>RadioButton</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_6">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>100</width>
           <height>30</height>
          </rect>
         </property>
         <attribute name="label">
          <string>sort out</string>
         </attribute>
         <widget class="QComboBox" name="comboBox_3">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>141</width>
            <height>22</height>
           </rect>
          </property>
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="tabletTracking">
           <bool>false</bool>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="currentText">
           <string>criteria</string>
          </property>
          <property name="duplicatesEnabled">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>criteria</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>value ++</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>value --</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>date</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>recent</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>oldest</string>
           </property>
          </item>
         </widget>
         <widget class="QPushButton" name="pushButton_11">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>10</y>
            <width>93</width>
            <height>28</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 255);</string>
          </property>
          <property name="text">
           <string>check</string>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
